src/esploader.ts

```diff
@@ -1140,6 +1146,11 @@ export class ESPLoader {
    * @returns {ROM} The Chip ROM
    */
   async runStub(): Promise<ROM> {
+    if (this.syncStubDetected) {
```

Comment:
Can we verify that the version of the stub that's running is the same as the version that we would upload ?

Reply:
Not according to what the stub_flasher code does here
https://github.com/espressif/esptool/blob/master/flasher_stub/stub_flasher.c#L240

Reply:
Fair, this follows the esptool.py approach 👍

Reply:
Any chance this can be merged? This is the last piece of work that the ExpressLRS project needs for it's web-flasher to be on the mainline of esptool-js.